src/current/cockroachcloud/authorization.md

```diff
@@ -133,31 +129,22 @@ This role can be granted at the scope of the organization, on an individual clus
 
 ### Folder Admin
 
-{{site.data.alerts.callout_success}}
-{% include_cached feature-phases/limited-access.md %}
-{{site.data.alerts.end}}
-
-This role is available only when your organization is enrolled in the [Folders]({% link cockroachcloud/folders.md %}) Limited Access.
+Folder Admins can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder.
 
-Folder Admins can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder. If granted on a specific folder, the role is inherited by descendent folders.
-
-An [Org Administrator](#org-administrator) role can grant any user or service account the Folder Admin role.
+An organization's initial [Org Administrator](#org-administrator) is automatically granted the Folder Admin role. Any user with the Org Administrator role can grant any user or service account the Folder Admin role.
 
 To create a cluster in a folder, the user must also have the Cluster Administrator or [Cluster Creator](#cluster-creator) role on that folder. To delete a cluster, the user must have the Cluster Administrator role, either on the cluster directly or by inheritance.
 
-This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder.
```

Comment:
How close does this render to the previous line.  It strikes me as odd that its not attached to the previous paragraph given that it starts with "This role".

Reply:
Oh interesting, I see.  This is under the "Folder Admin" section.  I was confused because the previous paragraph is talking about other roles.  If you think this ends up being clear enough, I'm good with that.